source/docs/guides/settlement.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Settlement Report
+layout: tutorial
+
+summary: >
+    A guide for Partners to integrate directly to the Settlement Report API.
+
+navigation:
+  header: na.tocs.na_nav_header
+  footer: na.tocs.na_nav_footer
+  toc: false
+  header_active: Guides
+---
+
+# Settlement Report
+
+## Report Overview
+
+As a Bambora Merchant, the Settlement Report provides an overview of
+transactions and upcoming settlement disbursement. You will be able to view
+pending settlement, settlement amount, when to expect it, and what fees were
+removed. With the settlement data, you can spend less time on manual tasks and
+reconciliation.
+
+As a Bambora Partner, you can help your customers reconcile by
+displaying settlement data within your software by using the Settlement Report.
+It allows you to display to your customers their pending settlement, settlement
+amount, when they can expect it, and what fees were removed. With the settlement
+data, your customers can spend less time on manual tasks and reconciliation.
+
```

Comment:
This seems to repeat the above. What about separating out the shared and the as a user parts (which I like)

The settlement report allows you to display to your customers their pending settlement, settlement
amount, when they can expect it, and what fees were removed. With the settlement
data, you and your customers can spend less time on manual tasks and reconciliation.

As a Bambora Merchant, the Settlement Report provides an overview of
transactions and upcoming settlement disbursement.

As a Bambora Partner, you can help your customers reconcile by
displaying settlement data within your software by using the Settlement Report.

Reply:
:shrug: That copy is entirely Matt's words.